client/src/components/home/Home.js

```diff
@@ -1,13 +1,57 @@
-import { useSelector } from "react-redux";
-import { selectUser } from "../../features/auth/auth.selectors";
+import SeriesCard from "./SeriesCard";
+import { MDBRow} from 'mdb-react-ui-kit';
+import { useEffect, useState } from 'react';
+import axios from 'axios';
 
 const Home = () => {
-    const user = useSelector(selectUser);
 
+  const [recommendedSeries, setRecommendedSeries] = useState([]);
+
+  useEffect(() => {
```

Comment:
```
useEffect(() => {
    seriesService.getCommonAmongFollowing(...) // move axios.get there
       .then(response => setRecommendedSeries(response.data.whereverTheSeriesAre)
       .catch(console.error)
  }, []);
```

Also, use the existing config file for `http://localhost:8080` value.
Same applied to mostPopular `useEffect`